### DISCUSSION TITLE
Usunięcie guest account @outlook z tenanta

### DISCUSSION THREAD

**Discussion by dzejzipl** [AzureCommunityPL/group#152] [Pomoc] [answered]
Hej,
Ma ktoś pomysł jak usunąć konto @outlook.com z czyjegoś tenanta?
![2020-11-13_10-53-17](https://user-images.githubusercontent.com/13648944/99195394-a35f5680-2785-11eb-8ead-f0b2cd1104d3.png)
Niestety, odpada kontakt z adminem tegoż tenanta, a trochę psuje mi to pracę. 
Próbowałem poprzez https://myaccount.microsoft.com/organizations?leave=true aczkolwiek.. Niestety nie przeszło logowanie, bo tam się logujemy poprzez work or school account.

**Comment by smajchrz** [comment, marked as answer]
To może być to:

https://docs.microsoft.com/en-us/azure/active-directory/external-identities/leave-the-organization

**Reply by dzejzipl** [reply]
Ot. To przeszło. 
Dziękuję bardzo! :)

**Reply by kaluzaaa** [reply]
@dzejzipl jak działa to korzystaj z magicznego przycisku Mark as answer :)

**Reply by dzejzipl** [reply]
> @dzejzipl jak działa to korzystaj z magicznego przycisku Mark as answer :)

Zrobione Szefie! ;)

**Reply by eklime** [reply]
Warto jednak mieć na uwadze, że ten mechanizm działa tylko wtedy, gdy jesteście w stanie zalogować się tym kontem do tamtego tenanta. Jeśli ktoś po tamtej stronie wyłączy Wam "Sign-in allowed" to *upa. Nie usuniecie się sami z tenanta :D
![image](https://user-images.githubusercontent.com/12485979/99243169-929ff680-2800-11eb-9111-29a4f530f678.png)
